doc/migrations/6.1.0_6.2.0.md

```diff
@@ -0,0 +1,13 @@
+## Internal databases
+
+So far MongooseIM used Mnesia to replicate the in-memory data between cluster nodes.
+Now there is an option to use [CETS](https://github.com/esl/cets/) instead, and all internal databases **must** be listed in a new section called [`internal_databases`](../../configuration/internal-databases).
+
+In order to continue using Mnesia, you **have to** configure it explicitly in `mongooseim.toml`:
+
+```toml
+[internal_databases]
+  [internal_databases.mnesia]
+```
+
+You can always decide to switch to CETS in the future.
```

Comment:
Could write that  `[internal_databases]` default value is no databases.

Reply:
There is a link to the doc, and that one should contain the details.